vagrant/provision/once-as-root.sh

```diff
@@ -57,14 +52,20 @@ info "Configure NGINX"
 sed -i 's/user www-data/user vagrant/g' /etc/nginx/nginx.conf
 echo "Done!"
 
-info "Enabling site configuration"
+info "Enable site configuration"
 ln -s /app/vagrant/nginx/app.conf /etc/nginx/sites-enabled/app.conf
 echo "Done!"
 
-info "Initailize databases for MySQL"
+info "Initialize databases for MySQL"
 mysql -uroot <<< "CREATE DATABASE yii2advanced"
 mysql -uroot <<< "CREATE DATABASE yii2advanced_test"
 echo "Done!"
 
+info "Enable remote access(root user) to MySQL from 192.168.83.1"
+mysql -uroot <<< "GRANT ALL PRIVILEGES ON yii2advanced.* TO 'root'@'192.168.83.1'  WITH GRANT OPTION"
+mysql -uroot <<< "GRANT ALL PRIVILEGES ON yii2advanced_test.* TO 'root'@'192.168.83.1'  WITH GRANT OPTION"
+echo "Done!"
+
```

Comment:
IP is configurable: https://github.com/yiisoft/yii2-app-advanced/blob/master/vagrant/config/vagrant-local.example.yml#L16

It mustn't be hardcoded.